common/meta/types/proto/request.proto

```diff
@@ -0,0 +1,59 @@
+// Copyright 2021 Datafuse Labs.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package meta;
+
+message SeqV {
+  uint64 seq = 1;
+  bytes data = 2;
+}
+
+// Get request and response
+message TxnGetRequest {
+  string key = 1;
+}
+
+message TxnGetResponse {
+  string key = 1;
+
+  optional SeqV value = 2;
+}
+
+// Put request and response
+message TxnPutRequest {
+  string key = 1;
+  bytes value = 2;
+  // if or not return the prev value
+  bool prev_kv = 3;
+}
+
+message TxnPutResponse {
+  string key = 1;
+  optional SeqV prev_value = 2;
+}
+
+// Delete request and response
+message TxnDeleteRequest {
+  string key = 1;
+  // if or not return the prev value
+  bool prev_kv = 2;
+}
+
+message TxnDeleteResponse {
+  string key = 1;
+  bool success = 2;
+  optional SeqV prev_value = 3;
```

Comment:
In the request, the flag is `prev_kv`, which looks like it is expecting a key-value pair. But the response has only the field `prev_value`.

Maybe rename `Request.prev_kv` to `prev_value`?

Reply:
done